test/e2e/framework/framework.go

```diff
@@ -103,11 +103,6 @@ type Framework struct {
 	// should abort, the AfterSuite hook should run all Cleanup actions.
 	cleanupHandle CleanupActionHandle
 
-	// afterEaches is a map of name to function to be called after each test.  These are not
-	// cleared.  The call order is randomized so that no dependencies can grow between
```

Comment:
should we worry about moving from randomised to ordered?

Reply:
I don't think so. We are not breaking anything ("ordered" is a valid order when previously only "randomized" was guaranteed) and I am not worried about such dependencies actually occurring in practice. Even if they do, the Ginkgo model of deterministic execution would make them okay.